app/src/main/res/values/strings.xml

```diff
@@ -670,11 +670,9 @@
 
     <!-- Inactive tabs -->
     <!-- Category header of a preference that allows a user to enable or disable the inactive tabs feature -->
-    <string name="preferences_inactive_tabs">Inactive tabs on tab tray</string>
-    <!-- Title for inactive tabs preference -->
-    <string name="preferences_inactive_tabs_title">Make tabs inactive after 14 days</string>
-    <!-- Summary of inactive tabs preference -->
-    <string name="preferences_inactive_tabs_summary">To improve speed, tabs will be moved into the inactive tabs section on your tab tray</string>
+    <string name="preferences_inactive_tabs">Move old tabs to inactive</string>
+    <!-- Title of inactive tabs preference -->
+    <string name="preferences_inactive_tabs_title">Tabs you haven\'t viewed for two weeks get moved to the inactive section.</string>
```

Comment:
@rocketsroger it's late and I may be missing something, but why the backslash in `haven\'t`?

Reply:
(seems like we don't escape anywhere else, but land when ready)

Reply:
Confirmed that we do need it.  Will not build without the backslash.  I'll land this.

Reply:
Fixed.  Thanks